docs/upgrade-notes/data-migrations-rds.md

```diff
@@ -0,0 +1,171 @@
+---
+id: data_migrations_rds
+title: Running data migrations for RDS
+hide_title: false
+---
+
+## Background
+
+This release of Cumulus (x.x.x) integrates with RDS and creates a new PostgreSQL database for archiving Cumulus data (e.g. granules, files, executions).
+
+While eventually Cumulus will only support using a PostgreSQL-compatible database as its data archive, for now the system will perform **parallel writes** to both DynamoDB and PostgreSQL so that all new data is archived in both datastores.
+
+However, in order to copy all of your previously written data from DynamoDB to PostgreSQL, you will need to run data migration scripts that we have provided and which this document will explain how to use.
+
+## Upgrade steps
+
+Follow the steps outlined below in precisely this order to upgrade your deployment and run the data migrations.
+
+### 1. Deploy a new RDS cluster
+
+Cumulus deployments require an Aurora [PostgreSQL 10.2](https://www.postgresql.org/) compatible database to be provided in addition to the existing DynamoDB/ElasticSearch backend with the eventual goal of utilizing the PostgreSQL database as the primary data store for Cumulus.
+
+> **NOTE**: Users are *strongly* encouraged to plan for and implement a database solution that scales to their use requirements, meets their security posture and maintenance needs and/or allows for multi-tenant cluster usage.
+
+Refer to the docs on [how to deploy a new RDS cluster](./../deployment/postgres-database-deployment.md).
+
+### 2. Deploy your data-persistence module
+
+The following new variables have been added:
+
+- `vpc_id`
+- `permissions_boundary_arn`
+- `rds_user_access_secret_arn`
+- `rds_security_group_id`
+
+You will need to update your data-persistence module to include these new variables related to RDS. See the configuration in our template-deploy repo for reference: <https://github.com/nasa/cumulus-template-deploy/tree/master/data-persistence-tf>
+
+Then you can re-deploy your data-persistence module as usual:
+
+```bash
+terraform apply
+```
+
+### 3. Deploy and run data-migration1
+
+Navigate to the directory `data-migration1-tf` and copy the following `.example` files:
```

Comment:
We need to tell them to create this directory first, following the example of the template-deploy erpo

Reply:
[`cd8beaf` (#2195)](https://github.com/nasa/cumulus/pull/2195/commits/cd8beaf54511f6945fed65a51f35bf24fc8a69fc)